quickwit-cluster/src/cluster.rs

```diff
@@ -75,30 +87,64 @@ impl Member {
     }
 }
 
-impl TryFrom<NodeId> for Member {
```

Comment:
I've always found the `Member` name that we inherited from artillery vague. Can we go with `ClusterMember` or `ClusterPeer`?

Reply:
True, I would go for `ClusterMember`